### PR TITLE
[build infra] focal to use openjdk 11 instead of 17 to get rid of false errors

### PR DESCRIPTION
**Pull request by Jimexist**
focal to use openjdk 11 instead of 17 to get rid of false errors

- [x] Did you create an [Apache Jira](https://issues.apache.org/jira/projects/THRIFT/issues/) ticket?  (not required for trivial changes)
- [x] If a ticket exists: Does your pull request title follow the pattern "THRIFT-NNNN: describe my issue"?
- [x] Did you squash your changes to a single commit?  (not required, but preferred)
- [x] Did you do your best to avoid breaking changes?  If one was needed, did you label the Jira ticket with "Breaking-Change"?
- [x] If your change does not involve any code, include `[skip ci]` anywhere in the commit message to free up build resources.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -6,7 +6,7 @@ Build-Depends: dotnet-runtime-6.0, dotnet-sdk-6.0, debhelper (>= 9), build-essen
     pkg-config, libtool, bison, flex, libboost-dev | libboost1.56-dev | libboost1.63-all-dev,
     python-all, python-setuptools, python-all-dev, python-all-dbg,
     python3-all, python3-setuptools, python3-all-dev, python3-all-dbg,
-    openjdk-8-jdk | openjdk-8-jdk-headless | openjdk-11-jdk | openjdk-11-jdk-headless | default-jdk,
+    openjdk-8-jdk | openjdk-8-jdk-headless | openjdk-11-jdk | openjdk-11-jdk-headless | openjdk-17-jdk | openjdk-17-jdk-headless | default-jdk,
```
Comment (emmenlau) [inline debian/control:9]: Do we even still need `openjdk-8-jdk` and `openjdk-8-jdk-headless`, or could they be dropped at the same time? Just asking, I'm not in the details!

Comment (Jimexist) [reply debian/control:9]: i think it can be dropped once we move away from xenial and welcome focal and have both focal and bionic available

Comment (emmenlau) [reply debian/control:9]: Ok, thanks for checking!

**Comment by Jimexist** [comment]
I'm merging this as this has now fixed the jdk issue. the next thing up is the swift error.

**Pull request merged**